### DISCUSSION TITLE
Link a doc (invoice, receipt) PDF

### DISCUSSION THREAD

**Discussion by Picaland** [fattureincloud/api#355] [Q&A] [answered]
Salve a tutti,
Non riesco a trovare nelle API una documentazione per ottenere il link al PDF di un documento (invoice, receipt).
per intenderci es https://compute.fattureincloud.it/doc/*.pdf
Qualcuno sa darmi quelche indicazione ?

Grazie!

**Comment by emajo** [comment, marked as answer]
Ciao @Picaland, per ottenere il link dei pdf è necessario fare una chiamata GET del documento interessato, a quel punto il link si troverà all'interno del campo `url`.

Grazie,
Emanuele.

**Reply by Picaland** [reply]
Ciao @emajo 
Quindi questo https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/issued_documents/-document_id-

Get Issued Document
get /c/{company_id}/issued_documents/{document_id} 

Grazie per la risposta mi era sfuggito questo

- url: string┃null -> [Temporary] [Read Only] Issued document url of the document PDF file
👍
